Signal/translations/en.lproj/Localizable.strings

```diff
@@ -10,7 +10,8 @@
 /* Label for 'invite' button in contact view. */
 "ACTION_INVITE" = "Invite to Signal";
 
-/* Label for 'sent message' button in contact view. */
+/* Label for 'sent message' button in contact view.
```

Comment:
s/sent/send

And this comment doesn't provide very useful context, maybe "Button label on a contact share message"

Also might want to introduce a unique string, since we're using this in multiple places, or pull into CommonStrings

Reply:
> s/sent/send

Done.

> Also might want to introduce a unique string, since we're using this in multiple places, or pull into CommonStrings

I'm inclined not to introduce a new string, given that both places we're using have very similar semantics (Send a text message to a user) and the exact same copy, in order to minimize the burden on our translators and leverage our existing translations.  I wish Transifex defaulted to existing translations for strings with the same EN copy.